From: Sebastian Ramacher <sramacher@debian.org>
Date: Tue, 31 Jul 2018 05:18:20 +0200
Subject: Do not install MIT HRTF data

---
 CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index fd32f7f..e005dc5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,10 +36,6 @@ list(APPEND headers
     include/hrtf/hrtf.h
     include/hrtf/mit_hrtf.h
     include/hrtf/sofa_hrtf.h
-    include/normal/mit_hrtf_normal_44100.h
-    include/normal/mit_hrtf_normal_48000.h
-    include/normal/mit_hrtf_normal_88200.h
-    include/normal/mit_hrtf_normal_96000.h
     source/kiss_fft/kiss_fftr.h
     source/kiss_fft/kiss_fft.h
 )
